cosmo_tester/test_suites/cli/linux_test.py

```diff
@@ -44,11 +49,159 @@ def test_cli_install_flow_linux(linux_cli_tester, logger):
 def get_linux_image_settings():
     return [
         ('centos_7', 'rhel_centos_cli_package_url', 'rpm'),
-        ('ubuntu_14_04', 'debian_cli_package_url', 'deb'),
+        # ('ubuntu_14_04', 'debian_cli_package_url', 'deb'),
         ('rhel_7', 'rhel_centos_cli_package_url', 'rpm'),
     ]
 
 
+def test_cfy_logs_linux(linux_cli_tester, logger):
+    cli_host = linux_cli_tester['cli_host']
+    example = linux_cli_tester['example']
+    paths = linux_cli_tester['paths']
+
+    _prepare(cli_host.run_command, example, paths, logger)
+    _test_cfy_logs(cli_host.run_command, cli_host, example, paths, logger)
+
+
+def test_tmux_session_linux(linux_cli_tester, logger):
+    cli_host = linux_cli_tester['cli_host']
+    example = linux_cli_tester['example']
+    paths = linux_cli_tester['paths']
+
+    _prepare(cli_host.run_command, example, paths, logger)
+    _test_tmux_session(cli_host.run_command,
+                       example.manager.run_command,
+                       example,
+                       paths,
+                       logger)
+
+
+def test_cfy_logs_linux_cluster(linux_cluster_cli_tester, logger):
+    cli_host = linux_cluster_cli_tester['cli_host']
+    manager = linux_cluster_cli_tester['manager']
+    paths = linux_cluster_cli_tester['paths']
+
+    linebreak = '\\r\\n' if 'C:\\' in paths['cfy'] else '\n'
```

Comment:
This is in the linux tests, so it definitely doesn't need to check whether it's linux. Either way, hopefully we can use json output- and if we do need this function in multiple places let's factor it out into one place.